packages/coq/coq.8.12.0/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+maintainer: "coqdev@inria.fr"
+authors: "The Coq development team, INRIA, CNRS, and contributors."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "git+https://github.com/coq/coq.git"
+license: "LGPL-2.1-only"
+synopsis: "Formal proof management system"
+description: """
+The Coq proof assistant provides a formal language to write
+mathematical definitions, executable algorithms, and theorems, together
+with an environment for semi-interactive development of machine-checked
+proofs. Typical applications include the certification of properties of programming
+languages (e.g., the CompCert compiler certification project and the
+Bedrock verified low-level programming library), the formalization of
+mathematics (e.g., the full formalization of the Feit-Thompson theorem
+and homotopy type theory) and teaching.
+"""
+
+depends: [
+  "ocaml" {>= "4.05.0" & < "4.12"}
+  "ocamlfind" {build}
+  "num"
+  "conf-findutils" {build}
+]
+build: [
+  [
+    "./configure"
+    "-configdir" "%{lib}%/coq/config"
+    "-prefix" prefix
+    "-mandir" man
+    "-docdir" doc
+    "-libdir" "%{lib}%/coq"
+    "-datadir" "%{share}%/coq"
+    "-coqide" "no"
+    "-flambda-opts" "-O3 -unbox-closures"
```

Comment:
Having this on by default seems a bit too dangerous — there's serious enough bugs that 8.12.0 doesn't fix — some with bad ocaml versions, others in combo with native-compiler.
The best way forward seems to make native-compiler depend on a "package flag", encoded as a dummy package.

For the last discussion see https://github.com/ocaml/opam-repository/pull/16779 and https://coq.zulipchat.com/#narrow/stream/237663-coq-community-devs.20.26.20users/topic/flambda-using.20Docker-Coq.20images, but I think that's the summary.

Reply:
Sure I can remove this. I took it from the core-dev packages.

Reply:
If those flags cause problems, IMO they should be removed from the core-dev packages as well.